app/views/internal/broadcasts/index.html.erb

```diff
@@ -19,30 +19,29 @@
   </div>
 </div>
 
-<div class="row my-3">
-  <div class="col">
-    <div class="alert alert-warning"><strong>Note: You must ensure that your Broadcast is active in order for it to be sent to users!<strong></div>
-    <div class="list-group-flush w-100">
-      <% @broadcasts.each do |broadcast| %>
-        <a class="list-group-item list-group-item-action flex-column align-items-start" href="/internal/broadcasts/<%= broadcast.id %>/edit">
-          <h3 class="mb-2">Title: <%= broadcast.title %></h3>
-          <h3>Broadcast Type: <%= broadcast.type_of %></h3>
-          <h3>Content:</h3>
-          <p>
-            <%= broadcast.processed_html %>
-          </p>
-          <h3>Last Active On: <%= broadcast.active_status_updated_at&.strftime("%b %d, %Y %H:%M UTC") %></h3>
-          <h3>Status:</h3>
-          <h3>
-            <span class="badge badge-<%= broadcast.active? ? "success" : "warning" %>">
+<div class="alert alert-warning"><strong>Note: You must ensure that your Broadcast is active in order for it to be sent to users!</strong></div>
+
+<table class="table">
```

Comment:
I made a decision to pull the "/internal/broadcasts" index into a table while working on this feature. While our "/internal" pages don't have the most consistent styling, refactoring the Broadcasts index into a table is more closely follows the "/internal" styling convention. Additionally, this refactor makes it easier to add things to the Broadcasts index in the future and features like sorting will be easier to implement with this change.

Reply:
I LOVE that table!

Reply:
Thank you so much @mstruve! 😊